hack/build-images.sh

```diff
@@ -87,6 +90,27 @@ image openshift/hello-openshift              examples/hello-openshift
 docker build --no-cache -t openshift/deployment-example:v1 examples/deployment
 docker build --no-cache -t openshift/deployment-example:v2 -f examples/deployment/Dockerfile.v2 examples/deployment
 
+# images for integrated services (e.g. logging and metrics)
+clonedir="$(mktemp -d 2>/dev/null || mktemp -d -t clonedir.XXXXXX)"
+pushd $clonedir
+# logging
+git clone --depth 1 --branch ${OS_LOGGING_BRANCH} https://github.com/openshift/origin-aggregated-logging.git 
```

Comment:
We do have a process for this - pulling in multiple source repos.  The
question is whether that's appropriate here - if we are centralizing
build duties maybe each of those repos should have its own build
step... But we do need a central release pivot point, which includes
tagging those repos appropriately.